shadows/framework/src/main/java/org/robolectric/shadows/ShadowBitmap.java

```diff
@@ -721,6 +721,22 @@ public void setCreatedFromResId(int resId, String description) {
     appendDescription(" for resource:" + description);
   }
 
+  void setPixelsForcibly(
```

Comment:
I think a cleaner approach would be to rename the have the following two methods:

1) setPixelsInternal, which has the same functionality as the current setPixels:
``` 
    void setPixelsInternal(
      int[] pixels, int offset, int stride, int x, int y, int width, int height) {
    this.colors = pixels;
  }
```

2) The `@Implementation` setPixels:
```
  @Implementation
  protected void setPixels(
      int[] pixels, int offset, int stride, int x, int y, int width, int height) {
    checkBitmapMutable();
    setPixelsInternal(pixels, offset, stride, x, y, width, height);
  }
```

Then ShadowBitmapFactory can call setPixelsInternal directly